app/controllers/api/v1/ratings_controller.rb

```diff
@@ -11,7 +11,9 @@ def show
   end
 
   def create
-    rating = Rating.new(user_id: params[:user], skill_id: params[:skill], score: params[:score])
+    rating = Rating.new(rating_params)
+    rating.user = User.where(id: session[:user]['id']).first
+    rating.skill = Skill.where(id: params['rating']['skill']).first
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.